auth/service.go

```diff
@@ -130,9 +132,9 @@ func (svc service) Issue(ctx context.Context, token string, key Key) (Key, strin
 	case APIKey:
 		return svc.userKey(ctx, token, key)
 	case RecoveryKey:
-		return svc.tmpKey(recoveryDuration, key)
+		return svc.tmpKey(svc.loginDuration, key)
 	default:
-		return svc.tmpKey(loginDuration, key)
+		return svc.tmpKey(svc.loginDuration, key)
```

Comment:
Please remove the constant that is no longer used for this duration.

Reply:
Done